tests/test_utils.py

```diff
@@ -0,0 +1,15 @@
+import sys
+from girder_worker.utils import TeeStdOutCustomWrite, TeeStdErrCustomWrite, JobManager
+
+
+def test_TeeStdOutCustomeWrite(capfd):
+    _nonlocal = {'data': ''}
```

Comment:
The python convention for avoiding reserved words is to use a suffix _, so `nonlocal_` rather than `_nonlocal`.

(I don't actually care, just feeling pedantic.)